spec/functional/knife/ssh_spec.rb

```diff
@@ -36,17 +36,6 @@
     allow(Net::SSH).to receive(:configuration_for).and_return(ssh_config)
   end
 
-  # Force log level to info.
-  around do |ex|
-    old_level = Chef::Log.level
-    begin
-      Chef::Log.level = :info
-      ex.run
-    ensure
-      Chef::Log.level = old_level
-    end
-  end
```

Comment:
These tests didn't output anything when run alone so it didn't seem worth keeping this.

Reply:
yeah thats weird looking, if it didn't break the tests that's probably fine.